solution/backend/cmcs_regulations/settings/local.py

```diff
@@ -31,6 +31,9 @@
 AWS_ACCESS_KEY_ID = os.environ.get("FILE_MANAGER_AWS_ACCESS_KEY_ID", None)
 AWS_SECRET_ACCESS_KEY = os.environ.get("FILE_MANAGER_AWS_SECRET_ACCESS_KEY", None)
 AWS_STORAGE_BUCKET_NAME = os.environ.get("FILE_MANAGER_AWS_BUCKET_NAME", None)
+TEXTRACT_KEY_ID = os.environ.get("TEXTRACT_KEY_ID", None)
+TEXTRACT_SECRET_KEY = os.environ.get("TEXTRACT_SECRET_KEY", None)
```

Comment:
ideally our normal aws stuff would have access but it doesnt so you need to use a key that has acces too textract